.gitignore

```diff
@@ -33,6 +33,10 @@ xcuserdata
 timeline.xctimeline
 playground.xcworkspace
 
+# AppCode
+.idea
+.idea/
+
```

Comment:
Yes, these are meta files generated by AppCode (don't want to have them in the repo).